src/main/java/de/focusshift/zeiterfassung/feedback/FeedbackService.java

```diff
@@ -1,9 +1,11 @@
 package de.focusshift.zeiterfassung.feedback;
 
 import de.focusshift.zeiterfassung.tenancy.user.EMailAddress;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.ApplicationEventPublisher;
 import org.springframework.stereotype.Service;
 
+@ConditionalOnProperty(prefix = "zeiterfassung.feedback", name = "enabled", havingValue = "true")
 @Service
```

Comment:
Diese Beans könnte man nun auch in der FeedbackConfiguration herausgeben, falls das property passt. Dann hat man alles an einem Fleck.